src/ilasm/writer.cpp

```diff
@@ -982,6 +982,61 @@ BOOL Assembler::EmitEventsProps(Class* pClass)
     return ret;
 }
 
+HRESULT Assembler::AllocateStrongNameSignature()
+{
+    HRESULT             hr = S_OK;
+    HCEESECTION         hSection;
+    DWORD               dwDataLength;
+    DWORD               dwDataOffset;
+    DWORD               dwDataRVA;
+    VOID               *pvBuffer;
+    AsmManStrongName   *pSN = &m_pManifest->m_sStrongName;
+
+    // pSN->m_cbPublicKey is the length of the m_pbPublicKey
+    int keySize = (int)pSN->m_cbPublicKey;
+
+    if (keySize == 0)
+    {
+        dwDataLength = 0;
```

Comment:
I didn't look very hard into that, but I think that it likely cannot. We had that condition in the Roslyn code, so I brought it over.